src/pgduckdb.cpp

```diff
@@ -27,52 +33,79 @@ _PG_init(void) {
 		ereport(ERROR, (errmsg("pg_duckdb needs to be loaded via shared_preload_libraries"),
 		                errhint("Add pg_duckdb to shared_preload_libraries.")));
 	}
+
 	DuckdbInitGUC();
 	DuckdbInitHooks();
 	DuckdbInitNode();
 	DuckdbInitBackgroundWorker();
 }
 } // extern "C"
 
-/* clang-format off */
+static void
+DefineCustomVariable(const char *name, const char *short_desc, bool *var) {
```

Comment:
Nice helpers! Can you add the context and flags as an optional parameter though? Those are often necessary to set.